README.md

```diff
@@ -290,36 +290,37 @@ The attenuations (i.e. UCAN output, or "caveats") MUST be an array of heterogene
 
 This array MUST contain some or none of the following:
 1. A strict subset (attenuation) of the capability scope from the `prf` field
-2. Capabilities composed from multiple witnesses (see rights amplification)
+2. Capabilities composed from multiple proofs (see rights amplification)
 3. Capabilities originated by the `iss` DID (i.e. by parenthood)
 
-This array also includes time ranges and the witnesses that start latest and end soonest from the lower and upper time bounds of the range.
+This array also includes time ranges and the proofs that start latest and end soonest from the lower and upper time bounds of the range.
 
-The attenuation field MUST contain an array of JSON objects, which MAY be empty. A JSON capability MUST include the `with` and `can` fields and MAY have additional fields needed to describe the capability.
+The attenuation field MUST contain an array of JSON objects, which MAY be empty. A JSON capability MUST include the `with` and `can` fields, and MAY additionally have additional fields needed to describe the capability.
 
 ``` json
 {
   "with": $RESOURCE,
-  "can": $ABILITY
+  "can": $ABILITY,
+  "ext": $EXTENSION
```

Comment:
This is extremely minor, but `ext` feels a bit out of pace next to the more natural language English `with` and `can`. `dtl` (details) is the other one that comes up, but also technically these are all "details". I'll loop back to this later.

Reply:
FYI @Gozala